packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -102,11 +111,23 @@ export class HelmChart extends CoreConstruct {
       throw new Error('Helm chart timeout cannot be higher than 15 minutes.');
     }
 
+    if (!props.chart && !props.chartAsset) {
+      throw new Error('Helm chart or chartAsset must be specified');
+    }
+
+    if (props.chartAsset && (props.repository || props.version)) {
+      throw new Error(
+        'Helm repository and version cannot be used with chartAsset',
```

Comment:
```suggestion
        `Neither 'repository' nor 'version' can be used when configuring 'chartAsset'`,
```

Reply:
What about `props.chart`? It also cannot be used when `chartAsset` is passed. 
Also - these validations deserve a few tests as well.